subprojects/kotlin-dsl/src/main/kotlin/org/gradle/kotlin/dsl/NamedDomainObjectContainerExtensions.kt

```diff
@@ -41,6 +41,21 @@ inline operator fun <T : Any, C : NamedDomainObjectContainer<T>> C.invoke(
 }
 
 
+/**
+ * Allows the container to be configured via an augmented DSL.
+ *
+ * @param configuration The expression to configure this container with
+ * @return The container.
+ */
+@Suppress("nothing_to_inline")
+@Deprecated(level = DeprecationLevel.HIDDEN, message = "For backward compatibility only")
```

Comment:
❓ Shouldn't we set a removal version and nag? We don't want to carry over such members indefinitely.

Reply:
See my suggestion in https://github.com/gradle/gradle/pull/15514

Reply:
Merged. Thanks!